.github/workflows/scripts/setup_bundler

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# This script sets up the bundler versions we need for our older unit tests
+# and sets up the bundler config for when we are using the older mysql version
```

Comment:
Only 2 of the semicolons in this script are needed - the ones that smoosh `if` and `then` into a one-liner. The others are all redundant with the newline on the end of each line already signifying an ending.

```shell
if [[ SOMETHING == "value" ]]; then
                             ^-- keep these ones
```

Reply:
Ahh that makes sense, I originally had most of these all squished on one line in the ci.yml as steps, and then forgot about it once I moved this into its own file. Got it updated!